rootdir/etc/init.qcom.rc

```diff
@@ -463,7 +463,6 @@ service ril-daemon2 /system/bin/rild -c 2
     socket rild2 stream 660 root radio
     socket rild-debug2 stream 660 radio system
     user root
-    disabled
```

Comment:
I rather leave this disabled as I fear it may cause issues with single sim models, but testing will needed to be conducted. it looks like ril-daemon2 should be started by init.class_main.sh https://github.com/AICP/device_motorola_clark/blob/mm6.0/rootdir/etc/init.class_main.sh#L63